src/mpcCoreKit.ts

```diff
@@ -86,6 +86,8 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
 
   private enableLogging = false;
 
+  private accountIndex;
```

Comment:
Can we try move this to the coreKit's state?
so that refresh will retain the previous account index set